howto/stream/exchange-oob-data.md

```diff
@@ -23,6 +24,8 @@ The following instructions guide you to exchange OOB data using a specific imple
 
 The following instructions will walk you through how to set up data channels and perform data transmission in both directions between an Android application and a WebRTC platform.
 
+Before proceeding, ensure that you have set up a web-based streaming client using the Anbox Cloud streaming stack. If you haven't done so, please follow our setup [guide](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/stream-client/).
+
 ### Prepare your web application
```

Comment:
We should make the code changes below based on top of what the streaming tutorial has as an example so it's simple for people to apply. Currently it's unclear for example where `stream.sendData` should be put.

Reply:
We should also point that out beforehand that we base all example code on top of this.

Reply:
+1